priv/repo/migrations/20210215195133_add_user_oauths.exs

```diff
@@ -0,0 +1,12 @@
+defmodule Recognizer.Repo.Migrations.AddUserOauths do
+  use Ecto.Migration
+
+  def change do
+    create table(:user_oauths) do
+      add :service, :string
+      add :service_guid, :string
+
+      add :user_id, references(:users, type: "int(11) unsigned", on_delete: :nothing)
+    end
+  end
```

Comment:
In our other migrations that touch existing data, we include a comment so we know to comment out the migration when we run stuff. Also something to consider, the current `service` type is `enum('facebook','github','google')`. We should probably change the migration to follow that for future use.

```suggestion
  use Ecto.Migration

  #  This migration is used for our existing database. It's commented out because
  #  we will eventually move authentication to it's own database and this code
  #  can be deleted.
  #
  #  def change do
  #    #
  #  end

  def change do
    create table(:user_oauths) do
      add :service, :"enum('facebook','github','google')"
      add :service_guid, :string

      add :user_id, references(:users, type: "int(11) unsigned", on_delete: :nothing)
    end
  end
```

Reply:
This has been updated :+1:

Reply:
sounds good. I've updated the migration to use the enum. Good call.